committee-code-of-conduct/README.md

```diff
@@ -32,16 +32,27 @@ The [charter](charter.md) defines the scope and governance of the Code of Conduc
 
 The members and their terms are as follows:
 
-### Term ends on August 7, 2020
-- Jaice Singer Dumars (Google)
-- Jennifer Rondeau (Stripe)
-- Carolyn Van Slyck (Microsoft)
-
 ### Term ends on August 7, 2021
+
 - Aeva van der Veen (Microsoft)
 - Tasha Drew (VMWare)
 
+### Term ends on August 7, 2022
+
+- Karen Chu (Microsoft)
+- Celeste Horgan (CNCF)
+- Tim Pepper (VMWare)
```

Comment:
Nit I guess, but if you happen to be amending this for another reason could I trouble you to change "VMWare" to "VMware" (and I guess also for Tasha above too)?  Otherwise I can submit a separate fixup later.